tests/ltp/README.md

```diff
@@ -714,7 +714,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/mknod/mknod09 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/mknodat/mknodat01 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/mknodat/mknodat02 | 0 | 0 | 0 |
-| /ltp/testcases/kernel/syscalls/mlock/mlock01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/mlock/mlock01 | 0 | 1 | 1 |
```

Comment:
Did mlock01 test on ext2fs pass in the past?

Reply:
Same as bind03, there was another PR to remove it